backend/api/urls.py

```diff
@@ -18,4 +19,10 @@
     path('', include(router.urls)),
     path('auth/signup/', SignUp.as_view()),
     path('login/', LoginView.as_view(), name="login"),
+    path(r'auth/users/reset_password_confirm/<uid>/<token>/',
```

Comment:
1. POST http://127.0.0.1:8000/api/auth/users/reset_password/
```json
{
    "email": "rofel@rofel.ru"
}
```

2. Результат
```text
Пожалуйста, перейдите на эту страницу и введите новый пароль:
http://127.0.0.1:8000/auth/users/reset_password_confirm/MTI/c168g3-b9c7b9d4874301e379ffa0744da24e94
Your username, in case you've forgotten: rofel@rofel.ru
```

3. GET http://127.0.0.1:8000/auth/users/reset_password_confirm/MTI/c168g3-b9c7b9d4874301e379ffa0744da24e94 

404 Not Found

мб я что-то не так сделал?